client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -193,6 +193,14 @@ function MetadataDropdownDirective() {
                 _.extend(scope.item, o);
                 scope.change({item: scope.item});
             };
+
+            $timeout(function() {
+                if (scope.list && scope.field === 'place') {
+                    scope.places = _.groupBy(scope.list, function(place) {
+                        return place.country === 'Australia' ? 'Australia' : 'Rest of World';
```

Comment:
We had a discussion about this. I will move it to vocabularies itself. Adding it there will give more flexibility in terms if tomorrow we want to group them into more than 2.